packages/editor/src/components/collab-sidebar/add-comment.js

```diff
@@ -45,6 +45,13 @@ export function AddComment( {
 		};
 	}, [] );
 	const blockElement = useBlockElement( clientId );
+	const { toggleBlockSpotlight } = unlock( useDispatch( blockEditorStore ) );
+
+	const unselectThread = () => {
+		setShowCommentBoard( false );
+		blockElement?.focus();
+		toggleBlockSpotlight( clientId, false );
+	};
```

Comment:
This is the process that occurs when adding a new note is canceled. The process for removing the spotlight state was missing.

Reply:
I think we also want to disable spotlight and call `toggleBlockSpotlight` `onBlur` event as well, since we're also closing the form on that event.